### PR TITLE
Set minimum macosx version to 10.8.

### PR DESCRIPTION
**Pull request by mgsergio**
@ygorshenin @syershov @mpimenov

### CODE REVIEW

File: common.pri [hunk 1]
```diff
@@ -2,6 +2,9 @@
 #
 # To use it, define ROOT_DIR variable and include($$ROOT_DIR/common.pri)
 
+# We use some library features that was introduced in macosx 10.8.
```
Comment (mpimenov) [inline common.pri]: s/was/were/

Comment (mpimenov) [inline common.pri]: s/macosx/Mac OS X/

File: common.pri [hunk 2]
```diff
@@ -2,6 +2,9 @@
...
+# Qt5.6.x Sets target osX version to 10.7 which leads to compile errors.
```
Comment (mpimenov) [inline common.pri]: s/Sets/sets/

**Comment by mpimenov** [comment]
LGTM after fix

File: common.pri (hunk 2 above)
Comment (ygorshenin) [inline common.pri]: Setting target os x version to 10.7 leads to compile error?

File: common.pri (hunk 1 above)
Comment (mgsergio) [inline common.pri]: Done.

Comment (mgsergio) [inline common.pri]: Done.

File: common.pri (hunk 2 above)
Comment (mgsergio) [inline common.pri]: Done.

**Comment by mpimenov** [comment]
Add a todo to remove this once new qt reaches homebrew?

File: common.pri (hunk 2 above)
Comment (mgsergio) [inline common.pri]: Yep

**Comment by mgsergio** [comment]
PTAL

File: common.pri [hunk 3]
```diff
@@ -2,6 +2,10 @@
 #
 # To use it, define ROOT_DIR variable and include($$ROOT_DIR/common.pri)
 
+# We use some library features that ware introduced in Mac OS X 10.8.
```
Comment (mpimenov) [inline common.pri]: s/ware/were/

Comment (mgsergio) [inline common.pri]: Done.

**Comment by mpimenov** [comment]
LGTM

File: common.pri [hunk 4]
```diff
@@ -2,6 +2,10 @@
 #
 # To use it, define ROOT_DIR variable and include($$ROOT_DIR/common.pri)
 
+# We use some library features that were introduced in Mac OS X 10.8.
+# Qt5.6.x sets target OS X version to 10.7 which leads to compile errors.
+# TODO: Remove when qt5.7 is available in brew.
```
Comment (ygorshenin) [inline common.pri]: s/qt/Qt/

Comment (ygorshenin) [inline common.pri]: will be available

Comment (ygorshenin) [inline common.pri]: Why do we need Qt5.7 from brew? It's possible to install it directly, without brew.

Comment (syershov) [inline common.pri]: Unnecessary todo, this is required project option, and we can manually control it.

File: common.pri [hunk 5]
```diff
@@ -2,6 +2,10 @@
...
+QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.8
```
Comment (syershov) [inline common.pri]: s/10.8/10.9/

File: common.pri (hunk 4 above)
Comment (mpimenov) [inline common.pri]: http://speakspeak.com/resources/english-grammar-rules/conjunctions/as-soon-as

Comment (mpimenov) [inline common.pri]: This one is even closer: http://dictionary.cambridge.org/ru/%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0/%D0%B1%D1%80%D0%B8%D1%82%D0%B0%D0%BD%D1%81%D0%BA%D0%B0%D1%8F-%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0/when

**Comment by mgsergio** [comment]
DUDE JTTP

**Comment by mgsergio** [comment]
PTAL

File: common.pri (hunk 4 above)
Comment (mgsergio) [inline common.pri]: Done.

File: common.pri (hunk 5 above)
Comment (mgsergio) [inline common.pri]: Done.

**Comment by ygorshenin** [comment]
LGTM

**Pull request merged**